wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/Button.java

```diff
@@ -79,7 +91,9 @@ public Button whileHeld(final Command command) {
    * @param toRun the runnable to run
    * @param requirements the required subsystems
    * @return this button, so calls can be chained
+   * @deprecated Use {@link #whileTrue(Command)} and construct a RunCommand manually
```

Comment:
Does this give the same behavior, or does it also need a RepeatCommand?

Reply:
For lambdas, the correct way to get the same behavior would be replacing the InstantCommand with a RunCommand.
 (Wrapping an InstantCommand in a RepeatCommand would call `initialize` once more immediately intuitive.)